src/components/Footer.jsx

```diff
@@ -9,7 +9,8 @@ function Footer() {
             <a
               href="#!"
               type="button"
-              className="rounded-full border-2 border-white text-white leading-normal uppercase hover:bg-black hover:bg-opacity-5 focus:outline-none focus:ring-0 transition duration-150 ease-in-out w-9 h-9 m-1">
+              className="rounded-full border-2 border-white text-white leading-normal uppercase hover:bg-black hover:bg-opacity-5 focus:outline-none focus:ring-0 transition duration-150 ease-in-out w-9 h-9 m-1"
+            >
```

Comment:
Is this generated?

Reply:
That's tailwind CSS

Reply:
Line 12 is part of Tailwind CSS.

I believe line 13 is the closing tag for it.

Reply:
Ah makes sense to me, we can resolve these then